lib/generators/administrate/routes/routes_generator.rb

```diff
@@ -20,15 +20,15 @@ def insert_dashboard_routes
       end
 
       def warn_about_invalid_models
-        namespaced_models.each do |invalid_model|
-          puts "WARNING: Unable to generate a dashboard for #{invalid_model}."
-          puts "         Administrate does not yet support namespaced models."
-        end
-
-        models_without_tables.each do |invalid_model|
-          puts "WARNING: Unable to generate a dashboard for #{invalid_model}."
-          puts "         It is not connected to a database table."
-          puts "         Make sure your database migrations are up to date."
+        invalid_dashboard_models.each do |model|
+          puts "WARNING: Unable to generate a dashboard for #{model}."
+          if namespaced_models.include?(model)
+            puts "       - Administrate does not yet support namespaced models."
```

Comment:
Perhaps a different question to ask is: Do we still need this warning? In general, we do support namespaced models. What's missing from the current support which means we should still have this warning?

Reply:
Exactly. I got here because I was looking into that. The error messages were confusing, so I thought of fixing that first, then looking into generating those namespaces. My thinking is that getting this to work properly may (perhaps) take long, so meanwhile let's at least make this more bearable, which is low-hanging fruit.

Reply:
Alright, let's do it!